ceph-mon/lib/charms.ceph_metrics/requirements.txt

```diff
@@ -0,0 +1,7 @@
+# Requirements for ceph_metrics dependencies
```

Comment:
Why are we vendoring this library? Did something change in charmcraft that prevents us from doing what was originally proposed?

Reply:
The vendored in part was something i staged by mistake. However, this approach is a bit problematic and I am marking this change closed in favor of #39